tachyon/crypto/commitments/kzg/kzg.h

```diff
@@ -114,6 +114,8 @@ class KZG {
 #endif
 
   void ResizeBatchCommitments(size_t size) {
+    // WARN(batzor): When resizing to larger size, the last values will be
+    // garbage and should to be filled with commitment results.
```

Comment:
Please fix for all instances of this comment
```suggestion
    // WARN(batzor): When resizing to a larger size, the last values will be
    // garbage and should be filled with commitment results.
```